source/features/faster-reviews.tsx

```diff
@@ -0,0 +1,75 @@
+import './faster-reviews.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import fetchDom from '../helpers/fetch-dom';
+import onReplacedElement from '../helpers/on-replaced-element';
+
+async function addSidebarReviewButton(): Promise<void> {
+	const reviewDetailsDropdown = (
+		<details className="rgh-faster-reviews details-reset details-overlay js-dropdown-details d-inline text-gray-dark">
+			<summary className="btn-link muted-link" data-hotkey="g r">review</summary>
+		</details>
+	);
+
+	select('[aria-label="Select reviewers"] .discussion-sidebar-heading')!.append(
+		<span style={{fontWeight: 'normal'}} className="js-reviews-container">
+			– {reviewDetailsDropdown}
```

Comment:
Rather than loading the whole form, it should just be a link to it, this would avoid your issue number 1

```js
const prFilesUrl = new URL(location.href);
prFilesUrl.pathname += '/files';
prFilesUrl.hash = 'submit-review'
```
```suggestion
			– <a href={prFilesUrl.href}>review</a>
```

Reply:
Oh, I misunderstood the issue. This behavior is much easier to implement and maintain of course :)